### PR TITLE
remave post-install from knative eventing kafka next

### PR DESCRIPTION
**Pull request by devguyio**
as per title

### CODE REVIEW

File: ci-operator/config/openshift-knative/eventing-kafka/openshift-knative-eventing-kafka-release-next__47.yaml [hunk 1]
```diff
@@ -57,14 +57,6 @@ images:
       - destination_dir: .
         source_path: /go/bin/mtreceive_adapter
   to: knative-eventing-kafka-mtreceive-adapter
-- dockerfile_path: openshift/ci-operator/knative-images/post-install/Dockerfile
-  from: base
-  inputs:
-    bin:
-      paths:
-      - destination_dir: .
-        source_path: /go/bin/post-install
```
Comment (matzew) [inline ci-operator/config/openshift-knative/eventing-kafka/openshift-knative-eventing-kafka-release-next__47.yaml:66]: we have the image now here: https://github.com/knative-sandbox/eventing-kafka/tree/main/config/channel/post-install

that's for the channel. I think we should add that too (via the generator).  

PR for source migrator is pending

File: ci-operator/config/openshift-knative/eventing-kafka/openshift-knative-eventing-kafka-release-next__48.yaml [hunk 2]
```diff
@@ -250,8 +242,6 @@ tests:
         name: knative-eventing-kafka-mtcontroller
       - env: KNATIVE_EVENTING_KAFKA_MTRECEIVE_ADAPTER
         name: knative-eventing-kafka-mtreceive-adapter
-      - env: KNATIVE_EVENTING_KAFKA_POST_INSTALL
```
Comment (matzew) [inline ci-operator/config/openshift-knative/eventing-kafka/openshift-knative-eventing-kafka-release-next__48.yaml:253]: I think I'd name that than `KNATIVE_EVENTING_KAFKA_CHANNEL_POST_INSTALL` ?

Comment (devguyio) [reply ci-operator/config/openshift-knative/eventing-kafka/openshift-knative-eventing-kafka-release-next__48.yaml:253]: that was removed anyway.

File: ci-operator/config/openshift-knative/eventing-kafka/openshift-knative-eventing-kafka-release-next__47.yaml (hunk 1 above)
Comment (devguyio) [reply ci-operator/config/openshift-knative/eventing-kafka/openshift-knative-eventing-kafka-release-next__47.yaml:66]: yes but this is a different version migrator. I can add that in a separate PR and I think we won't need an image for it. Just the manifest. In this PR I'm removing Travis post-install.

Comment (devguyio) [reply ci-operator/config/openshift-knative/eventing-kafka/openshift-knative-eventing-kafka-release-next__47.yaml:66]: I can see there's this image https://github.com/knative-sandbox/eventing-kafka/blob/main/config/channel/post-install/storage-version-migrator.yaml#L38 , okay. I thought there is a generally built one that we can directly address.

Ok, I'll add that new image then.

/hold

Comment (matzew) [reply ci-operator/config/openshift-knative/eventing-kafka/openshift-knative-eventing-kafka-release-next__47.yaml:66]: > let's wait until there is the source- image... it's a bit tricky


HOWEVER. You are correct with removing _this_ image :smile: 

/unhold
/lgtm
/approve

**Pull request merged**